src/main/steps/prl-cases/citizenSequence.ts

```diff
@@ -12,4 +17,14 @@ export const citizenSequence: Step[] = [
     showInSection: Sections.AboutEdgeCase,
     getNextStep: () => DASHBOARD_URL,
   },
+  {
+    url: PIN_ACTIVATION_ENTER_PIN_URL,
+    showInSection: Sections.AboutEdgeCase,
+    getNextStep: () => PIN_ACTIVATION_CASE_ACTIVATED_URL,
+  },
+  {
+    url: PIN_ACTIVATION_CASE_ACTIVATED_URL,
```

Comment:
@giorgi-hmcts Either we can have this or new post controller added in the route.
Right now you have added both so it's redundant.